decidim-core/app/views/decidim/shared/_orders.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= icon "arrow-up-s-line" %>
   <span><%= t("#{i18n_scope}.label") %></span>
 </button>
-<div id="dropdown-menu-order" class="order-by">
+<div id="dropdown-menu-order" class="order-by <%= class_css %>">
```

Comment:
```suggestion
<div id="dropdown-menu-order" class="order-by <%= css_class %>">
```

Extra warning @rsdeus : this change all the order dropdown of decidim. Be sure to check proposals sorting etc.

Reply:
Done. Thanks :smile: